README.md

```diff
@@ -1,12 +1,4 @@
-# PHP GitHub API 2.0
-
-In 2.0 lib no longer uses guzzle 3.7, instead it has an HTTPlug abstraction layer. 
-
-For old version please check:
-
-* [branch](https://github.com/KnpLabs/php-github-api/tree/1.7)
-* [readme](https://github.com/KnpLabs/php-github-api/tree/1.7/README.md)
-* [docs](https://github.com/KnpLabs/php-github-api/tree/1.7/doc)
+# PHP GitHub API 3.0
```

Comment:
Why the `3.0`? This library is still in the `2.x` cycle

Reply:
More relevant information is rather *Uses GitHub API v3*

Most recent version of this package is already in [released](https://github.com/KnpLabs/php-github-api/releases), no need to duplicated it in README, moreover in `composer` time.

Reply:
👎 Should be reverted. This is the name of the package, not of the API.

Reply:
Version is not a name, but I see your points.

Let's make it *PHP GitHub API* then to prevent ambiguity.